vendor.conf

```diff
@@ -4,4 +4,5 @@ github.com/spf13/cobra 2df9a531813370438a4d79bfc33e21f58063ed87
 github.com/spf13/pflag e57e3eeb33f795204c1ca35f56c44f83227c6e6
 github.com/ryanuber/go-glob 256dc444b735e061061cf46c809487313d5b0065
 github.com/morikuni/aec 39771216ff4c63d11f5e604076f9c45e8be1067b
-github.com/inconshreveable/mousetrap 76626ae9c91c4f2a10f34cad8ce83ea42c93bb75
\ No newline at end of file
+github.com/inconshreveable/mousetrap 76626ae9c91c4f2a10f34cad8ce83ea42c93bb75
+github.com/mitchellh/go-homedir b8bc1bf767474819792c23f32d8286a45736f1c6
```

Comment:
Is this new dep Windows compatible, too?

Reply:
Yes it is see https://github.com/mitchellh/go-homedir/blob/master/homedir.go#L120